README.md

```diff
@@ -1 +1,160 @@
-# Rotations
+# Rotations.jl
+
+This package implements various rotation parameterizations and defines conversions between them.  All rotation variables are stored as as immutable types.
+
+This package assumes [active (right handed) rotations](https://en.wikipedia.org/wiki/Active_and_passive_transformation) wheere applicable.
+
+### Rotation Parameterizations
+
+1. **Rotation Matrix** `RotMatrix{T <: AbstractFloat}`
+
+    A 3 x 3 rotation matrix storing the rotation.  This is a typealias of a [FixedSizeArrays](https://github.com/SimonDanisch/FixedSizeArrays.jl) `Mat{3,3,T}`.  A rotation matrix `R` should have the property `I = R * R<sup>T</sup>`.
+
+
+
+2. **Arbitrary Axis Rotation** `AngleAxis{T <: AbstractFloat}`
+
+    A 4 element immutable array with fields `theta`, `x`, `y`, and `z` to store the rotation angle and axis of the rotation.  This is a typealias of a [FixedSizeArrays](https://github.com/SimonDanisch/FixedSizeArrays.jl) `FixedVectorNoTuple{4, T}`.
```

Comment:
Fixed

On 9 May 2016 at 13:20, Chris Foster <notifications@github.com> wrote:

> In README.md
> <https://github.com/FugroRoames/Rotations.jl/pull/2#discussion_r62447120>:
>
> > +
> > +This package implements various rotation parameterizations and defines conversions between them.  All rotation variables are stored as as immutable types.
> > +
> > +This package assumes [active (right handed) rotations](https://en.wikipedia.org/wiki/Active_and_passive_transformation) wheere applicable.
> > +
> > +### Rotation Parameterizations
> > +
> > +1. **Rotation Matrix** `RotMatrix{T <: AbstractFloat}`
> > +
> > +    A 3 x 3 rotation matrix storing the rotation.  This is a typealias of a [FixedSizeArrays](https://github.com/SimonDanisch/FixedSizeArrays.jl) `Mat{3,3,T}`.  A rotation matrix `R` should have the property `I = R * R<sup>T</sup>`.
> > +
> > +
> > +
> > +2. **Arbitrary Axis Rotation** `AngleAxis{T <: AbstractFloat}`
> > +
> > +    A 4 element immutable array with fields `theta`, `x`, `y`, and `z` to store the rotation angle and axis of the rotation.  This is a typealias of a [FixedSizeArrays](https://github.com/SimonDanisch/FixedSizeArrays.jl) `FixedVectorNoTuple{4, T}`.
>
> Out of date docs - it's not an FSA any more.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/FugroRoames/Rotations.jl/pull/2/files/fb3a7882257b93afed97cfaa9bf56669bbf410dc#r62447120>
>